### ISSUE TITLE
MAGCruiseLibraries の運用について

### ISSUE THREAD

**Issue by yuu-nkjm** [MAGCruise/MAGCruiseLibraries#1]
@guccyon 
このプロジェクトに気付きませんでした．言語グリッド系のレポジトリと切り離されている方が分かりやすいので，
ここにlibをまとめようと思います．

https://github.com/MAGCruise/MAGCruiseLibraries/tree/master/servicegrid 以下は， @guccyon さんが手元でコンパイルしてまとめたjarが入っているという認識でよろしいでしょうか？

**Comment by takawitter** [comment]
横から失礼します。
今言語グリッドの基礎的なライブラリはmaven centralから入手できるので、
mavenを使えれば非常に楽になります。
maven使わなくても、下記からダウンロードできます。
http://search.maven.org/#search%7Cga%7C1%7Cnet.servicegrid

**Comment by guccyon** [comment]
はい、手元でコンパイルしたものです。Gitリポジトリに置くことで git submodule を利用できて環境構築の自動化が簡単にする為でした。

mavenセントラルであれば gradle を使って管理でも良いかもしれませんね。composerと同じように利用できます。

**Comment by yuu-nkjm** [comment]
@takawitter さん， @guccyon さん， ありがとうございます．

mavenベースでやろうかなとも思いましたが，MAGCruiseBrokerがCoreをキックする
ところのクラスパスとかもややこしそうなので，諦めました．

現時点では「MAGCruiseのプロジェクトはMAGCruiseLibrariesからlibファイルを
ビルドバスに取り込む」という運用にしようと思います．
